report/schemas.api.md

```diff
@@ -1677,6 +1677,17 @@ export namespace PreviewType {
 // @alpha
 export type Profile = {
     avatars: Avatar[];
+    country: Country;

```

Comment:
Are all these going to be mandatory from now? Otherwise, we should make them optional (like the profession and the rest), using the `?`.